profile_collection/startup/9991-zmq.py

```diff
@@ -0,0 +1,3 @@
+from bluesky.callbacks.zmq import Publisher
+
+Publisher('localhost:5577', RE=xrun)
```

Comment:
should we put this few lines into ``collection`` profile? I guess ``xrun`` might not necessarily exist in ``analysis`` environment.

Reply:
I think they are in the collection profile, right?

Reply:
NVM. I thought all changes are to ``analysis``.

This PR looks good to me then!